Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -708,4 +721,12 @@ - (void)viewIssuesTapped {
   [self.detailViewDelegate didSelectViewIssuesForBook:self.book sender:self];
 }
 
+-(void)didFinishUIAccessibilityAnnouncementNotification:(NSNotification *)notification {
```

Comment:
is this called when the system has finished reading any UIAccessibility announcement, even unrelated ones? For example, you have this view on the Catalog tab, but then you moved to the My Books tab. 

And is it possible for the message to be read twice?